### PR TITLE
gnome-backgrounds: update to 43.

### PR DESCRIPTION
**Pull request by oreo639**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/gnome-backgrounds/template [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 revision=1
 build_style=meson
-hostmakedepends=gettext
+hostmakedepends="gettext"
+depends="librsvg webp-pixbuf-loader"
 short_desc="Set of background images for the GNOME Desktop"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later, CC-BY-2.0, CC-BY-SA-2.0, CC-BY-SA-3.0"
 homepage="https://gitlab.gnome.org/GNOME/gnome-backgrounds"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
```
Comment (paper42) [inline srcpkgs/gnome-backgrounds/template:12]: replace $pkgname with gnome-backgrounds

**Comment by paper42** [comment]
image
This doesn't work with gnome-shell 42, let's merge this with GNOME 43.

**Comment by oreo639** [comment]
I never installed gnome-shell 43, is gdk-pixbuf installed?

**Comment by paper42** [comment]
Hmm, you are right, a restart helped.

**Pull request merged**